padertorch/contrib/examples/speech_enhancement/simple_mask_estimator/README.md

```diff
@@ -0,0 +1,48 @@
+Simple Mask Estimator
+=============
+
+This directory contains scripts to train and evaluate a simple mask estimator
+inspired by .
+
+Results
+-------
+
+The simple mask estimator model can achieve the following results:
+
+
+data type         |  pesq         | stoi         |      sdr 
+:------------------|--------------|--------------|--------------:
+observed           |  1.07       |  0.672       |  -0.79 dB
+masked             |  1.22       |  0.736       |  5.68 dB
+beamformed         |  1.91       |  0.958       |   17.10 dB  
+
+Training
+--------
+
+A storage root must be set with `export STORAGE_ROOT=/path/to/your/storage`.
+After installing `padertorch`, a training can for example be started with
+
+```bash
+$ STORAGE_ROOT=/path/to/your/storage; python -m padertorch.contrib.examples.speech_enhancement.simple_mask_estimator.train
+```
+
+Evaluation
+----------
+
+The evaluation requires `dlp_mpi` and `pb_bss` as additional dependencies.
+`dlp_mpi` can be installed via `pip install dlp_mpi` and `pb_bss` is available at [github.com/fgnt/pb_bss](github.com/fgnt/pb_bss).
+The evaluation can be started by
+
+```bash
+$ STORAGE_ROOT=/path/to/your/storage; python -m padertorch.contrib.examples.speech_enhancement.simple_mask_estimator.evaluate
+```
+It always evaluates the latest model in the specified STORAGE_ROOT
```

Comment:
Can you give an example, how to evaluate one specific model? I think the common use case is, that you want to evaluate one particular model.

Reply:
Or alternatively, can you mention here, that we dropped a CLI to keep this example as simple as possible?

Reply:
done